packages/rich-text-alpha/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@contentful/field-editor-rich-text-alpha",
```

Comment:
**Nit:** I'd prefer `@contentful/field-editor-rich-text-2-alpha` or `@contentful/field-editor-rich-text-next` as a wip name but it probably doesn't matter much.

Reply:
I get your reasoning. I prefer `next` of the two, `-2-alpha` is awkward :D

Reply:
Any reason not to use release channels? e.g. `alpha` or `next` under the current package name e.g. `@contentful/field-editor-rich-text@next` and later land with a major version bump when it's ready. That would mean using a different branch though e.g. `rich-text-v2` or something

Reply:
One advantage I see (in addition to being cleaner IMO) is that we don't have to change the `widgetId` everywhere when we are slowly rolling it out. I imagine `richTextEditor` being hard-coded in a couple of places that can be a bit of a hassle to adjust while being behind a feature flag.

Reply:
One nice thing about releasing it as its own widget is that we'll want to use a separate `widgetId` anyway when deciding which editor to choose in the webapp, Compose etc. Maybe there's a way around that I'm not thinking of though.

Reply:
> One nice thing about releasing it as its own widget is that we'll want to use a separate widgetId 

It could be because I'm not fully aware of how we plan to move forward on this but I actually see that as an issue (as explained in my previous comment). My understanding is that there won't be any major changes to the _public_ API shape, so in a way, this package is a drop-in replacement - but incomplete as of now- for the current `@contentful/field-editor-rich-text` and can be used (again, not complete yet) wherever `richTextEditor` widget is used, is that right?

If yes, then having a different `widgetId` might not work in our favour, because we will gradually roll out changes using the `alphaRichTextEditor` and then again gradually roll out replacing it with the actual `richTextEditor`.

Reply:
Well, if we decided to go with this approach we'd need to introduce some prop in `default-field-editors` to support it. I'm not opposed to it but also don't feel too strongly and don't feel like I get the complete vision here, happy to pair with you on this early next week or accept a PR to this PR.